### PR TITLE
25create conversation header

### PR DESCRIPTION
**Pull request by mikaGivner**
Hey Yishai,

I added a text file with  3clarification  sections 

Tnx

### CODE REVIEW

File: client/src/assets/logo.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (yishain11) [inline client/src/assets/logo.jsx:1]: remove all  empty lines in the file

File: client/src/assets/humburger.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import { SvgStyle } from "../style"
+export default function HumburgerSvg() {
+  
+    return (
+      <SvgStyle  width="24" height="20" viewBox="0 0 24 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1.83358C0 1.0972 0.596954 0.500244 1.33333 0.500244H22.6667C23.403 0.500244 24 1.0972 24 1.83358C24 2.56996 23.403 3.16691 22.6667 3.16691H1.33333C0.596954 3.16691 0 2.56996 0 1.83358ZM0 9.83358C0 9.0972 0.596954 8.50024 1.33333 8.50024H22.6667C23.403 8.50024 24 9.0972 24 9.83358C24 10.57 23.403 11.1669 22.6667 11.1669H1.33333C0.596954 11.1669 0 10.57 0 9.83358ZM0 17.8336C0 17.0972 0.596954 16.5002 1.33333 16.5002H22.6667C23.403 16.5002 24 17.0972 24 17.8336C24 18.57 23.403 19.1669 22.6667 19.1669H1.33333C0.596954 19.1669 0 18.57 0 17.8336Z" fill="white"/>
+      </SvgStyle>
+    )
+  }
+
```
Comment (yishain11) [inline client/src/assets/humburger.jsx:11]: remove all  empty lines in the file

File: client/src/assets/logo.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+export default function LogoSvg() {
+  
+  return (
+    <svg width="35" height="25" viewBox="0 0 35 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path d="M1.9165 1H33.2498V16.75C33.2498 18.6065 32.4245 20.387 30.9555 21.6997C29.4865 23.0125 27.494 23.75 25.4165 23.75H9.74984C7.67231 23.75 5.67987 23.0125 4.21083 21.6997C2.7418 20.387 1.9165 18.6065 1.9165 16.75V1Z" stroke="white" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round"/>
+    </svg>
+  )
+}
+
```
Comment (yishain11) [inline client/src/assets/logo.jsx:12]: remove all  empty lines in the file

File: client/src/pages/Foy_Yishai.txt [hunk 4]
```diff
@@ -0,0 +1,3 @@
+-[]- I have no the icon of the coffee from Tommer- just need to complete it.
```
Comment (yishain11) [inline client/src/pages/Foy_Yishai.txt:1]: remove this file

File: client/src/style/Header_style.jsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
+
+
+import styled from "styled-components";
+import "./index";
+export const HeaderStyle = styled.div`
+margin-top:0;
```
Comment (yishain11) [inline client/src/style/Header_style.jsx:6]: very good!

File: client/src/style/Header_style.jsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+export const HeaderStyle = styled.div`
+margin-top:0; 
+width:100%;
+ height:13rem;
+ display:flex;
+ padding:1rem;
+ background:#50924E;
+align-items:center;
+justify-content:space-between;
+font-size: 'poppins';
```
Comment (yishain11) [inline client/src/style/Header_style.jsx:14]: this is not font size, this is a font family

**Review by yishain11 — CHANGES_REQUESTED** [review]

File: client/src/pages/PageOne.page.jsx [hunk 7]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
```
Comment (yishain11) [inline client/src/pages/PageOne.page.jsx:1]: remove file

File: client/src/pages/PageTwo.page.jsx [hunk 8]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
```
Comment (yishain11) [inline client/src/pages/PageTwo.page.jsx:1]: remvoe file

File: client/src/style/Header_style.jsx [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+font-weight: 500;
+font-size: 1.5rem;
+color: #FFFFFF;
+
```
Comment (yishain11) [inline client/src/style/Header_style.jsx:18]: remove empty lines

File: client/src/style/svg_style.jsx [hunk 10]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (yishain11) [inline client/src/style/svg_style.jsx:1]: remvoe empty lines

**Review by yishain11 — CHANGES_REQUESTED** [review]
1. the width is too big, compare to figma design.
2. remove empty lines
3. remove pages and text - its good for testing, but in the final version we dont need it
4. you didnt fix the font size issue i pointed @ Header_style.jsx. plus - we use camelCase, not kabab_case. change file name

File: client/src/main.jsx [hunk 11]
```diff
@@ -1,14 +1,15 @@
 "use client";
 import { ErrorBoundary } from "react-error-boundary";
-import React from "react";
 import ReactDOM from "react-dom/client";
+import React from "react";
+import "./index.css";
```
Comment (yishain11) [inline client/src/main.jsx:5]: remove, there is no such file

File: client/src/components/Root.jsx [hunk 12]
```diff
@@ -0,0 +1,12 @@
+import { Outlet } from "react-router-dom";
+import Header from "./Header.jsx";
+import HamburgerSvg from "../assets/hamburger.jsx"
+import LogoSvg from "../assets/logo";
+export default function RootLayout() {
+  return (
+    <>
+      <Header leftIcon={<HamburgerSvg/>} midIcon={<LogoSvg/>} rightIcon={""}/>
```
Comment (yishain11) [inline client/src/components/Root.jsx:8]: use Glass.jsx (inside assets)

**Review by yishain11 — CHANGES_REQUESTED** [review]

File: client/src/pages/Intro.jsx [hunk 13]
```diff
@@ -1,21 +1,19 @@
-import { ThemeProvider } from "styled-components"
-import Glass from "../assets/Glass"
-import { Flex } from "../components/styles/Flex"
-
+import { ThemeProvider } from "styled-components";
+import Glass from "../assets/Glass";
+// import { Flex } from "../components/styles/Flex"
```
Comment (yishain11) [inline client/src/pages/Intro.jsx:3]: remove commented out code

**Review by yishain11 — CHANGES_REQUESTED** [review]

File: client/src/pages/Intro.jsx [hunk 14]
```diff
@@ -1,21 +1,19 @@
...
-  mobile: "798px"
-}
+  colors: {},
+  mobile: "798px",
+};
 
 const Intro = () => {
   return (
     <ThemeProvider theme={theme}>
-    <Flex>
```
Comment (yishain11) [inline client/src/pages/Intro.jsx:15]: why did you remove this?

**Review by yishain11 — CHANGES_REQUESTED** [review]

File: client/src/pages/Intro.jsx (hunk 14 above)
Comment (mikaGivner) [reply client/src/pages/Intro.jsx:15]: עשיתי קומפוננטה חדשה- קונבנציה לכל הדפים הלבנים שמתלבשים על בדף הלבן. הPagesStyle מחליף את הFlex ואפשר להוסיף את הסוג דיב הזה לכל מי שעבד על הדפים הלבנים
